crates/ruff_db/src/diagnostic/render.rs

```diff
@@ -60,13 +62,17 @@ impl std::fmt::Display for DisplayDiagnostic<'_> {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         if matches!(self.config.format, DiagnosticFormat::Concise) {
             match self.diag.severity() {
-                Severity::Info => f.write_str("info")?,
-                Severity::Warning => f.write_str("warning")?,
-                Severity::Error => f.write_str("error")?,
-                Severity::Fatal => f.write_str("fatal")?,
+                Severity::Info => write!(f, "{}", "info".bright_cyan().bold())?,
```

Comment:
We should respect the `config.color` option even for concise diagnostics similar to how it's done for the full diagnostic format here

https://github.com/astral-sh/ruff/blob/bd5b8a9ec6a1877c6d457c1f2e52e57ea37679de/crates/ruff_db/src/diagnostic/render.rs#L45-L49

and it would be great if we could reuse the same colors as for the full output format. Unfortunately, they're defined in the annotate_snippets crate. 

One option is to have our own `Stylesheet` struct in the `diagnostic` crate with a `plain` and `colored` (or `styled`) constructor with which we then initialize the renderer used in the full output format (by calling all the setter methods).

Reply:
Yeah for me, I think making sure we respect `config.color` is a blocker to merging this PR. That would make this a strict improvement I think.

But I do think that further synchronizing the colors can be done in another PR. And I also think it would be good to do. I'm not sure if we want to expose the ability for callers to set individual colors, but at least having two public profiles `styled` and `plain` makes sense to me. (I think this is the same thing you're saying @MichaReiser.)